src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/data/BehaviourSummary.kt

```diff
@@ -56,7 +77,7 @@ data class PrisonerIncentiveSummary(
   )
   val bookingId: Long,
   @Schema(description = "Internal reference for looking up the prisoners latest photo", example = "112121")
-  val imageId: Long,
+  val imageId: Long?,
```

Comment:
Oh, does this mean the `imageId` is optional then?

Reply:
yes